internal/vm/vsock.go

```diff
@@ -207,14 +207,19 @@ func tryConnect(logger *logrus.Entry, udsPath string, port uint32) (net.Conn, er
 		}
 	}()
 
-	err = tryConnWrite(conn, vsockConnectMsg(port), vsockConnectMsgTimeout)
+	msg := vsockConnectMsg(port)
+	err = tryConnWrite(conn, msg, vsockConnectMsgTimeout)
 	if err != nil {
-		return nil, vsockConnectMsgError{cause: err}
+		return nil, vsockConnectMsgError{
+			cause: errors.Wrapf(err, `failed to write %q within %s`, msg, vsockConnectMsgTimeout),
+		}
 	}
 
 	line, err := tryConnReadUntil(conn, '\n', vsockAckMsgTimeout)
 	if err != nil {
-		return nil, vsockAckError{cause: err}
+		return nil, vsockAckError{
+			cause: errors.Wrapf(err, `failed to read "OK <port>" within %s`, vsockAckMsgTimeout),
```

Comment:
is there a way to log the `port` number as well? or would we not want to do that?

Reply:
Let me take a look. I think the port here is assigned by Firecracker.